e2e/testcases/kcc_test.go

```diff
@@ -129,3 +135,93 @@ func TestKCCResourcesOnCSR(t *testing.T) {
 		nt.T.Fatal(err)
 	}
 }
+
+func TestResourceGroupKCC(t *testing.T) {
+	nt := nomostest.New(t, nomostesting.ACMController, ntopts.KCCTest)
+
+	ns := v1.Namespace{}
+	ns.Name = "resourcegroup-e2e"
+	nt.T.Cleanup(func() {
+		// all test resources are created in this namespace
+		if err := nt.KubeClient.Delete(ns.DeepCopy()); err != nil {
```

Comment:
DeepCopy is expensive. Maybe just make a var for the namespace name and recreate the Namespace for the create and delete.

Reply:
Done.